js/src/views/Settings/Parity/parity.js

```diff
@@ -94,18 +55,30 @@ export default class Parity extends Component {
             </div>
           </div>
           <div className={ layout.details }>
-            <LanguageSelector />
             { this.renderModes() }
+            <LanguageSelector />
+            <Features />
```

Comment:
Looks a bit strange, right in the middle between two features : language and logs. Would be better on top or on bottom